docs-ref-mapping/reference-unified.yml

```diff
@@ -356,11 +380,17 @@
         landingPageType: Service
         children:
         - com.microsoft.azure.cosmosdb*
+    - name: Spring Data
```

Comment:
This should be placed with the other `Spring` stuff.

https://github.com/Azure/azure-docs-sdk-java/blob/fc56192ae5ea8c8f82afbe96d4e52310d20c47b0/docs-ref-mapping/reference-unified.yml#L812

See where I mean? I don't think it goes under `cosmos`. You may have details I don't though!

Reply:
I did not find the best place to go. I found these packages are under sdk/cosmos, so I decided to put it here. The Spring boot and Spring cloud seem not the best place to go. I don't want to give it a new page. 

Let put it here for now. We can simply move to the right position if people complains.

Reply:
https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/cosmos

Reply:
Hmm. Alrighty. Definitely not blocking this PR on this piece of feedback.